shared/templates/file_permissions/oval.template

```diff
@@ -67,6 +67,11 @@
       #}}
       <filter action="include">state_file_permissions{{{ FILEID }}}_{{{ loop.index0 }}}_mode_not_{{{ FILEMODE }}}</filter>
     {{%- endif %}}
+      <filter action="exclude">exclude_symlinks_{{{ FILEID }}}</filter>
```

Comment:
Do we want to wrap this with an if as well? So that in cases that we can to check symlinks we can?

Reply:
sorry, I completely missed your comment.
I don't fully remember now, but the way this oval is written, it is not following symbolic links I think.
Also what would be a great if for this case? platform specific?

Reply:
I don't have any case off the top my head. Was just thought. I think we can leave it as is.